src/app/loginComponents/accounts/external/accounts.component.ts

```diff
@@ -238,4 +248,43 @@ export class AccountsExternalComponent implements OnInit, OnDestroy {
     this.ngUnsubscribe.next();
     this.ngUnsubscribe.complete();
   }
+
+  revokeToken() {
+    let dialogRef = this.dialog.open(RevokeTokenDialogComponent, {
```

Comment:
No need to change, but for future reference, in general prefer `const` over `let` -- only use `let` if you're going to modify the value -- it makes reading the code a little bit easier.